### PR TITLE
add elastic search post

### PR DESCRIPTION
**Pull request by mikepmunroe**

### CODE REVIEW

File: _posts/2015-05-18-elastic-search.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+product requries a lot of statistical analysis, you might work with
+[R](http://www.r-project.org/). Elasticsearch supports all of these different
+languages through clients built for each language. You can find a list of
+supported clients on the
+[Elastic website](https://www.elastic.co/guide/en/elasticsearch/client/community/current/clients.html).
+These clients will make it easier to integrate Elasticsearch with the
+programming language used to develop your application.
+
+#### Elasticsearch supports many features.
+Following support of many programming languages, Elasticsearch support many
```
Comment (davekaro) [inline _posts/2015-05-18-elastic-search.md]: _supports_

File: _posts/2015-05-18-elastic-search.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+*  full-text search - basic search matching words against documents
+*  multifield matching - advanced search matching multiple different searches
+against documents
+*  proximity matching - matching words close to each other, such as a phrase
+against documents
+*  partial matching - often used for matching against mispellings
+*  tweak relevancy - change the weighting of search criteria to make some more
+relevant than others
+*  multiple language support - French, German, Spanish, etc.
+*  Aggregations - ask sophisticated questions about our data in near real time
```
Comment (davekaro) [inline _posts/2015-05-18-elastic-search.md]: why are "Aggregations" and "Geolocation" capitalized?

File: _posts/2015-05-18-elastic-search.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+choice.
+
+#### Elasticsearch is fast.
+We have been using Elasticsearch on a few different client projects and have
+been very impressed with its performance.
+
+In one instance, Elasticsearch is being utilized for full-text search,
+multifield matching, and using its [percolator](https://www.elastic.co/guide/en/elasticsearch/reference/1.3/search-percolate.html)
+against millions of documents and it is instantaneous. Percolator is a larger
+subject than covered in this post that we are planning to discuss in a future,
```
Comment (davekaro) [inline _posts/2015-05-18-elastic-search.md]: _the future_ ... unless we are living multiple realities?!?!!!

File: _posts/2015-05-18-elastic-search.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+*  multifield matching - advanced search matching multiple different searches
+against documents
+*  proximity matching - matching words close to each other, such as a phrase
+against documents
+*  partial matching - often used for matching against mispellings
+*  tweak relevancy - change the weighting of search criteria to make some more
+relevant than others
+*  multiple language support - French, German, Spanish, etc.
+*  aggregations - ask sophisticated questions about our data in near real time
+*  geolocation - support location information
```
Comment (davekaro) [inline _posts/2015-05-18-elastic-search.md:46]: Maybe include "percolator" in this list? I was surprised in the next paragraph to read about the percolator... when that wasn't mentioned here

**Comment by davekaro** [comment]
Looks great.

File: _posts/2015-05-18-elastic-search.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+We have used it for many of these already and have been very happy with that
+choice.
+
+#### Elasticsearch is fast.
+We have been using Elasticsearch on a few different client projects and have
+been very impressed with its performance.
+
+In one instance, Elasticsearch is being utilized for full-text search,
+multifield matching, and using its [percolator](https://www.elastic.co/guide/en/elasticsearch/reference/1.3/search-percolate.html)
```
Comment (HeroicEric) [inline _posts/2015-05-18-elastic-search.md]: link is out of date https://www.elastic.co/guide/en/elasticsearch/reference/current/search-percolate.html

File: _posts/2015-05-18-elastic-search.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+tags :
+title: "3 Reasons Why You Should Try Elasticsearch"
+description: A small introduction to Elasticsearch.
+author: mike_munroe
+---
+{% include JB/setup %}
+
+Do you have an application or set of applications that require searching
+capabilities, has a large data set and you want to do some analysis on that
+data? If so, you should chech out Elasticsearch. I'm going to outline 3 reasons
```
Comment (HeroicEric) [inline _posts/2015-05-18-elastic-search.md]: "chech" -> "check"

File: _posts/2015-05-18-elastic-search.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+across companies and products. For example, if your product is a web
+application, you might utilize [Ruby on Rails](http://rubyonrails.org/), one of
+our frameworks of choice at [Greenfield](http://greenfieldhq.com). If your
+product requries a lot of statistical analysis, you might work with
+[R](http://www.r-project.org/). Elasticsearch supports all of these different
+languages through clients built for each language. You can find a list of
+supported clients on the
+[Elastic website](https://www.elastic.co/guide/en/elasticsearch/client/community/current/clients.html).
+These clients will make it easier to integrate Elasticsearch with the
+programming language used to develop your application.
```
Comment (HeroicEric) [inline _posts/2015-05-18-elastic-search.md]: I might add something in this section that also says that you can use elasticsearch with any language that allows you to send GET and POST requests. These clients just help you out a little bit

File: _posts/2015-05-18-elastic-search.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+#### Elasticsearch supports many features.
+Following support of many programming languages, Elasticsearch supports many
```
Comment (HeroicEric) [inline _posts/2015-05-18-elastic-search.md]: "Following support of many programming languages," can probably be taken out.

**Pull request merged**